de-de/haskell-de.html.markdown

```diff
@@ -212,7 +212,7 @@ foo 5 -- 60
 -- Haskell hat einen Operator `$`, welcher Funktionsapplikation durchführt.
 -- Im Gegenzug zu der Standard-Funktionsapplikation, welche linksassoziativ ist
 -- und die höchstmögliche Priorität von "10" hat, ist der `$`-Operator
--- rechtsassoziativ und hat die Priorität 0. Dieses hat (idr.) den Effekt,
+-- rechtsassoziativ und hat die Priorität 0. Dieses hat (i.d.R.) den Effekt,
```

Comment:
I might be wrong, but I think that's proper grammar. "i.d.R." stands for "in der Regel", (means "in general"). I think you shouldn't write "idr.", because first of all, it's three words, and second of all, the third word is capitalized if you don't abbreviate it. (I might be wrong; German isn't my native tongue)